### ISSUE TITLE
Design: Incorporate MSI in new /all UI

### ISSUE THREAD

**Issue by hoosteeno** [mozilla/bedrock#7265]
## Description

We want to include MSI installers in the new designs for /all. This is an interface design challenge. We need to explore it and mock it up. 

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Installer types, like MSI, are included in the /all interface somehow
- [ ] Mockups are ready for coding

#### Tasks
- [ ] This is an example of a task. Use a checklist!
- [ ] This is an example of a task.

**Comment by hoosteeno** [comment]
I met with Mark, who'll give us some direction on the UI here. Mark will follow up on this comment with final direction. Below are a couple drafts, not quite ready for coding.

The demo is at https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/all/.

We discussed possibly adding installer types to the OS dropdown, as in the attached images. From a design perspective, this would mean...

- [ ] Get final names for the things in the OS dropdown. [There appear to be MSI installers for Release/Beta/Dev/Nightly Windows 64](https://support.mozilla.org/en-US/kb/deploy-firefox-msi-installers?redirectlocale=en-US&redirectslug=firefox-customization-msi-installers). That would mean splitting the Win64 item into`Windows 64-bit [exe or something]` and `Windows 64-bit [msi or something]`
- [ ] Change the label on the dropdown (are these "platforms?" instead of OS? And you are no longer "using" the one you download, necessarily). Tyler Ahn can direct us on this. [Here's an artifact](https://docs.google.com/spreadsheets/d/1-eRoBpWwYUT5nwn7aPdiOEEa-c6pHvs3vlvwewWNJ1A/edit#gid=0) where we track this kind of content.
- [ ] Add an info modal for different installer types: At launch time, this box should include .exe and .msi types. In the near future it will have pkg and dmg types. This, too, requires copy, which should be obtained from Tyler and updated in the crib sheet linked above.

Is that everything?

![Screenshot_2019-06-03 Download the Firefox browser in English (US) and more than 90 other languages](https://user-images.githubusercontent.com/94519/58813916-fe2cde80-85e1-11e9-9c5e-30a7ab58e7cd.png)

<img width="1294" alt="Screen Shot 2019-06-03 at 9 22 54 AM" src="https://user-images.githubusercontent.com/94519/58813923-008f3880-85e2-11e9-9506-996d662e8507.png">

**Comment by mrkwvr** [comment]
Discussed this with Wendy and a few other team members and we all decided its best to add installers to the platform dropdown menu. I will follow up with Tyler to get copy and direction.

**Comment by alexgibson** [comment]
For the new modal copy, it might also be a good opportunity to spend some time explaining the differences between 32bit and 64bit builds?

**Comment by alexgibson** [comment]
The chosen hex color for the icon is `#0060DF`.

**Comment by skrelnick23** [comment]
Copy suggestions are located here: https://docs.google.com/document/d/1JrZWk2Ny75KQ_MeaBEny7S9nFqtrWSjAQKrgvmiNKXY/edit

Along with file extension definitions here:https://docs.google.com/spreadsheets/d/1-eRoBpWwYUT5nwn7aPdiOEEa-c6pHvs3vlvwewWNJ1A/edit#gid=475749008

**Comment by alexgibson** [comment]
Thank you @skrelnick23 

I'm going to work directly with @claychaffin to try and finalize this copy.

**Comment by claychaffin** [comment]
Copy deck https://docs.google.com/document/d/1phOeJaAiy5WlNSLiWKmlcMQiK5K22m1Fel8Uco3-7TE/edit?usp=sharing

**Comment by claychaffin** [comment]
Copy update: I pinged Tyler Ahn about the installers copy this afternoon and she is working to get the descriptions to us—a new tab in the spreadsheet— which I will then massage to make user-facing. I also reached out to Tim M about the capitalization of "Firefox Browser" as well as whether we'll refer to it as "Firefox Browser" (as with iPhone, iPad— no "the") or as "the Firefox Browser", as we have been. I will also consult with UX about best-practice/industry standard installer prompt copy. Thanks.
